labconnect/main/opportunity_routes.py

```diff
@@ -455,25 +458,20 @@ def filterOpportunities():
 #     abort(500)
 
 
-# # Jobs page
-# @main_blueprint.route("/getOpportunityCards", methods=["GET"])
-# def getOpportunityCards():
-#     if request.method == "GET":
-#         # query database for opportunity
-#         query = db.session.execute(
-#             db.select(Opportunities).where(Opportunities.active == True)
-#         )
-
-#         data = query.fetchall()
+# Jobs page
+@main_blueprint.get("/getOpportunityCards")
+def getOpportunityCards():
+    # query database for opportunity
+    query = db.session.execute(
+        db.select(Opportunities).where(Opportunities.active == True)
+    )
 
-#         # return data in the below format if opportunity is found
-#         cards = {
-#             "data": [packageOpportunityCard(opportunity[0]) for opportunity in data]
-#         }
+    data = query.fetchall()
```

Comment:
what is fetchall?

Reply:
It was already there when I started working on it. It returns the query as an iterable list of tuples.